### DISCUSSION TITLE
How to read debug json logs without the cruft

### DISCUSSION THREAD

**Discussion by thejan2009** [renovatebot/renovate#28021] [Suggest an Idea]
### Tell us more.

Hey,
we've recently enabled debug log file output while the default stdout is the pretty output at INFO level. Now we finally get the debug logs and they aren't truncated by the CI runner. The issue is that .ndjson is a bit hard to read, with all the `name`, `hostname`, `timestamp` and similar repeating on every line.

Do you use any helper commands to view the "useful" subset of every line in debug log?

**Comment by rarkins** [comment]
Pretty output is meant to be human readable, and the JSON output machine-readable. We don't currently have any tools to convert the machine readable back into human readable.

**Comment by jamietanna** [comment]
https://docs.renovatebot.com/community-tools/#renovate-pretty-log-tui is an option here
